src/main/java/core/basesyntax/AreaCalculable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculable {

```

Comment:
The interface name 'AreaCalculable' is using a verb form. Consider renaming it to something more noun-oriented, such as 'AreaCalculator', to clearly represent it as an entity.